### DISCUSSION TITLE
Cascading Delete

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#6255] [Q&A] [answered]
i need to delete user with related profiles:
```groovy
model USER {
id
profiles  Profile[]
}

model Profile {
id
email
phone
}
```

this.prisma.user.delete({where: {id: userId}, includes: {profiles :true}})
i got error :
```bash
  The change you are trying to make would violate the required relation 'User' between the `Profile` and `User` models.
```

**Comment by ruheni** [comment, marked as answer]
Hi @BishoyNarouz. 👋🏾

Alex here from the Prisma team.

Sadly, Prisma doesn't support cascade deletes. In the meantime, you can use [Paljs plugin](https://paljs.com/plugins/delete/) to perform cascade deletes. This feature is on our [roadmap](https://www.notion.so/prismaio/Prisma-Roadmap-50766227b779464ab98899accb98295f) and hopefully be released soon.

In case of any other inquiries, feel free to reach out. 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
